src/overlays/actors/ovl_En_Skb/z_en_skb.c

```diff
@@ -525,7 +533,7 @@ s32 EnSkb_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* p
     s16 pad[2];
 
     if (limbIndex == 11) {
-        if ((this->unk_283 & 2) == 0) {
+        if ((this->bodybreakState & 2) == 0) { // head limb, head is still attached
```

Comment:
like previously mentioned by others, I dont think this should named after BodyBreak when it isnt directly related to the BodyBreak system. Its more like it works in parallel with the body break system. all BodyBreak does is spawn the parts as they fall off. But this is more about controlling what gets drawn on the model that still exists

`breakFlags` or `bodyFlags` are fine for me, but using both terms together implies direct relationship with the system named BodyBreak